modules/phase_field/include/materials/GAMMAanisoGAUSS.h

```diff
@@ -0,0 +1,396 @@
+
+  ADMaterialProperty<Real> & _qzg;
+
+  ADMaterialProperty<Real> & _Ggammabar;
+
+  ADMaterialProperty<Real> & _Ggammamin2grains;
+
+  ADMaterialProperty<Real> & _TotGauss;
+
+  ADMaterialProperty<Real> & _MGBVALUE;
+
+  const Real _sigmaBASE;
+
+  const Real _GgammaBASE;
+
+  const Real _gammaBASE;
+
+  const Real _f0gammaBASE;
+
+  const Real _lgbBASE_minimum;
+
+  const Real _alphaswitch;
+
+  const Real _betaswitch;
+
+  unsigned int _libnum;
+
+  const Real _amplitudeScale;
+
+  const Real _sharpness;
+
+  const Real _Gaussian_Tolerance;
+
+  const FileName _Library_file_name;
+
+  const FileName _Quaternion_file_name;
+
+  const GrainTrackerInterface & _grain_tracker;
+
+  const Real _length_scale;
+
+  const Real _time_scale;
+
+  const Real _JtoeV;
+
+  const Real _GBMobility;
+
+  const Real _Mob;
+
+  const Real _Q;
+
+  const Real _T;
+
+  const Real _kb;
+
+  const Real _BoundaryNormal;
+
+  const unsigned int _op_num;
+
+  const std::vector<const ADVariableValue *> _vals;
+
+  const std::vector<const ADVariableGradient *> _grad_vals;
+
+  const MaterialPropertyName _Ggamma_name;
+  const MaterialPropertyName _Ggamma_EN_name;
+
+  const MaterialPropertyName _dGgammadx_name;
+  const MaterialPropertyName _dGgammadx_EN_name;
+
+  const MaterialPropertyName _dGgammady_name;
+  const MaterialPropertyName _dGgammady_EN_name;
+
+  const MaterialPropertyName _dGgammadz_name;
+  const MaterialPropertyName _dGgammadz_EN_name;
+
+  const MaterialPropertyName _dGgammadxplus_name;
+  const MaterialPropertyName _dGgammadxplus_EN_name;
+  const MaterialPropertyName _dGgammadyplus_name;
+  const MaterialPropertyName _dGgammadyplus_EN_name;
+  const MaterialPropertyName _dGgammadzplus_name;
+  const MaterialPropertyName _dGgammadzplus_EN_name;
+
+  const MaterialPropertyName _gamma_name;
+  const MaterialPropertyName _gamma_EN_name;
+
+  const MaterialPropertyName _S_switch_name;
+
+  const MaterialPropertyName _dgammadx_name;
+  const MaterialPropertyName _dgammadx_EN_name;
+
+  const MaterialPropertyName _dgammady_name;
+  const MaterialPropertyName _dgammady_EN_name;
+
+  const MaterialPropertyName _dgammadz_name;
+  const MaterialPropertyName _dgammadz_EN_name;
+
+  const MaterialPropertyName _dgammadxplus_name;
+  const MaterialPropertyName _dgammadxplus_EN_name;
+  const MaterialPropertyName _dgammadyplus_name;
+  const MaterialPropertyName _dgammadyplus_EN_name;
+  const MaterialPropertyName _dgammadzplus_name;
+  const MaterialPropertyName _dgammadzplus_EN_name;
+
+  const MaterialPropertyName _m_name;
+  const MaterialPropertyName _m_EN_name;
+
+  const MaterialPropertyName _kappa_name;
+  const MaterialPropertyName _kappa_EN_name;
+
+  const MaterialPropertyName _L_name;
+  const MaterialPropertyName _L_EN_name;
+
+  const MaterialPropertyName _lgb_name;
+  const MaterialPropertyName _lgb_EN_name;
+
+  unsigned int _grain_num;
+};
+
+// %gamma        g(gamma)    f_0_max(gamma)
+// %
+// gamma_function =
```

Comment:
as above

Reply:
That was fixed. See updated files. I deleted them.